Sources/MongoSwift/BSON/Document+Codable.swift

```diff
@@ -15,10 +15,11 @@ extension Document: Codable {
         var container = encoder.container(keyedBy: _BSONKey.self)
         for (k, v) in self {
             let key = _BSONKey(stringValue: k)!
-            if let val = AnyBSONValue(ifPresent: v) {
-                try container.encode(val, forKey: key)
-            } else {
+            let val = AnyBSONValue(v)
```

Comment:
nit: let's only bother creating this struct if we're gonna encode it (under the else condition)

Reply:
Done.